packages/admin/cms-admin/src/blocks/createImageLinkBlock.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { createImageLinkBlock } from "./createImageLinkBlock";
+import { InternalLinkBlock } from "./InternalLinkBlock";
+import { PixelImageBlock } from "./PixelImageBlock";
+
+describe("createImageLinkBlock", () => {
+    it("should allow overriding the name", () => {
```

Comment:
This test currently fails because of import errors when importing modules from node_modules. I tried mocking them, but that only caused more errors -> could we mock the node_modules completely?

Reply:
Errors for reference:

```
SyntaxError: Unexpected token 'export'

      11 | var _styles = require("@mui/styles");
      12 | var React = _interopRequireWildcard(require("react"));
    > 13 | var _reactDnd = require("react-dnd");
         |                 ^
      14 | var _Table = require("./Table");
      15 | var _TableBodyRow = require("./TableBodyRow");
      16 | var _excluded = ["moveRow", "onDragEnd", "dragHandleIcon", "classes"];
```

Reply:
When mocking react-dnd using:

```
jest.mock("react-dnd", () => {
    return;
});
```

```
SyntaxError: Unexpected token 'export'

      16 | import { styled } from "@mui/material/styles";
      17 | import makeStyles from "@mui/styles/makeStyles";
    > 18 | import { deepClone } from "@mui/x-data-grid/utils/utils";
         | ^
      19 | import * as React from "react";
      20 | import { FormattedMessage } from "react-intl";
      21 |
```

And so it goes on 😅

Reply:
looks like a ESM module problem,  a few packages are already ESM and not commonJs and can't be transformed :)
if thats the case you have a jest config called "transformIgnorePatterns" where you can take out those specific packages

for example
```transformIgnorePatterns: ["/node_modules/(?!(p-debounce|swiper|ssr-window|dom7|axios|pretty-bytes)/)"],```

Reply:
I tried that, but it doesn't work for me 😢